packages/web/src/components/gcds-footer/stories/gcds-footer.stories.tsx

```diff
@@ -65,11 +65,61 @@ const Template = (args) => (`
 </GcdsFooter>
 `).replace(/\s\snull\n/g, '');
 
+const TemplatePlayground = (args) => (`
+<!-- Web component code (Angular, Vue) -->
+<gcds-footer
+  ${args.display != "compact" ? `display="${args.display}"` : null}
+  ${args.contextualHeading && args.contextualLinks ? `contextual-heading="${args.contextualHeading}"` : null}
+  ${args.contextualHeading && args.contextualLinks ? `contextual-links='${args.contextualLinks}'` : null}
+  ${args.subLinks ? `sub-links='${args.subLinks}'` : null}
+  ${args.lang != "en" ? `lang="${args.lang}"` : null}
+>
+</gcds-footer>
+
+<!-- React code -->
+<GcdsFooter
+  ${args.display != "compact" ? `display="${args.display}"` : null}
+  ${args.contextualHeading && args.contextualLinks ? `contextualHeading="${args.contextualHeading}"` : null}
+  ${args.contextualHeading && args.contextualLinks ? `contextualLinks='${args.contextualLinks}'` : null}
+  ${args.subLinks ? `subLinks='${args.subLinks}'` : null}
+  ${args.lang != "en" ? `lang="${args.lang}"` : null}
+>
+</GcdsFooter>
+`).replace(/\s\snull\n/g, '');
+
 export const Default = Template.bind({});
 Default.args = {
   display: 'full',
   contextualHeading: 'Contextual navigation',
   contextualLinks: '{ "Why GC Notify": "#", "Features": "#", "Activity on GC Notify": "#" }',
   subLinks: '',
   lang: 'en'
+};
+
+export const DisplayCompact = Template.bind({});
+DisplayCompact.args = {
+  lang: 'en'
```

Comment:
Can you add `display: 'compact'` here? It's currently showing up as `display="undefined"` in the code preview. Or if `compact` is supposed to be the default value, can you add it as the default value? Right now we don't have a default defined for `display`.